crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -308,21 +313,37 @@ impl SceneSpawner {
                         .entry(handle.id())
                         .or_insert_with(HashSet::new);
                     spawned.insert(instance_id);
+
+                    if parent.is_none() {
```

Comment:
I'd prefer a comment here stating something to the effect of 

> // If there is no parent there is no more work to do, and therefore we can send a ready event.
> // Else a ready event will be sent later if the `NonExistentScene` branch is taken.

(which is what I assume is happening).

Would aid future readers not familiar with `bevy_scene` (like me!).

Reply:
Done.
Let me know if anything else is unclear!